packages/hardhat/test/Gift.test.js

```diff
@@ -101,8 +101,7 @@ describe('Gift', () => {
 
     it('reverts', async () => {
       await assertRevert(
-        Ethernauts.connect(owner).gift(owner.address),
-        'No more Ethernauts can be gifted'
+        Ethernauts.connect(owner).gift(owner.address)
```

Comment:
Is there a way to check also the reverted error?

Reply:
Not sure if i can do that.